data/strings/strings.txt

```diff
@@ -13610,32 +13615,36 @@
   [error_enter_correct_facebook_page]
     tags = android
     en = Enter a valid Facebook web address, account, or page name
-	fr = Saisissez une adresse web, un compte ou un nom de page Facebook valide
+	  fr = Saisissez une adresse web, un compte ou un nom de page Facebook valide
     pl = Wprowadź poprawny link, nazwę konta lub nazwę strony na Facebooku
+    es = Introduce una dirección web, una cuenta o un nombre de página de Facebook válidos
```

Comment:
Please sort languages here and below too. Only en goes first, every other translation should follow the alphabetical order.